ShareX.UploadersLib/ShareX.UploadersLib.csproj

```diff
@@ -795,11 +794,9 @@
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Import Project="$(SolutionDir)\.nuget\NuGet.targets" Condition="Exists('$(SolutionDir)\.nuget\NuGet.targets')" />
   <PropertyGroup>
-    <PreBuildEvent>cd $(ProjectDir)APIKeys\
-
-if not exist APIKeysLocal.cs (
-    type nul &gt; APIKeysLocal.cs
-)</PreBuildEvent>
+    <PreBuildEvent>cd $(ProjectDir)APIKeys
+echo=&gt;&gt; APIKeysLocal.cs
```

Comment:
@davidruhmann is this checking if we already have a APIKeysLocal.cs as before? We don't want APIKeysLocal.cs overwritten. Thanks.